example-program.tortoise

```diff
@@ -1,7 +1,13 @@
 P 2  # select pen 2
 D    # pen down
-W 2  # draw west 2cm
+D
+W 2  # draw west 2 cm
 N 1  # then north 1
-E 2  # then east 2
-S 1  # then back south
 U    # pen up
+U
```

Comment:
Let's not alter `example.tortoise`. That's the only example from the book; let's keep that as-is.